noodles-util/src/variant/format.rs

```diff
@@ -0,0 +1,17 @@
+/// A variant format.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum Format {
+    /// Variant Call Format (VCF).
+    Vcf,
+    /// BCF.
+    Bcf,
+}
+
+/// A variant compression.
+#[derive(Clone, Copy, Debug, Eq, PartialEq)]
+pub enum Compression {
```

Comment:
In noodles-cram, `BlockContentEncoderMap` encoders are `Option<Encoder>`, where `None` represents no compression. Should we take the same approach here, i.e., removing `Uncompressed` and having compression be an `Option<Compression>`?

Reply:
That's probably sensible! I've made the corresponding change.